### PR TITLE
addded support for sending pinMode through SIM

### PR DESCRIPTION
**Pull request by jordans6**

### CODE REVIEW

File: avionics/cmake-build-debug/FLARE.cbp [hunk 1]
```diff
@@ -0,0 +1,891 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jesseli2002) [inline avionics/cmake-build-debug/FLARE.cbp]: Was this file supposed to be added? `cmake-build-debug` sounds like temporary files

**Review by jesseli2002 — CHANGES_REQUESTED** [review]
Look over the list of files changed and remove the debug/temp files

File: avionics/cmake-build-debug/FLARE.cbp (hunk 1 above)
Comment (jordans6) [reply avionics/cmake-build-debug/FLARE.cbp]: Nope that's my bad.  I'll fix that

File: avionics/envs/x86/lib/HAL/pin_util.cpp [hunk 2]
```diff
@@ -5,6 +5,12 @@ namespace Hal {
 
 void pinMode(Pin pin, PinMode mode) {
     // TODO - Maybe report this somehow?
```
Comment (jesseli2002) [inline avionics/envs/x86/lib/HAL/pin_util.cpp]: TODO can be removed now; probably spacing can then be cleaned up

**Review by jesseli2002 — CHANGES_REQUESTED** [review]
sorry there's just one comment to update

**Review by jesseli2002 — APPROVED** [review]

**Pull request merged**